core/src/lib.rs

```diff
@@ -115,43 +151,67 @@ pub enum Team {
     Crew,
     Impostors,
 }
-#[derive(Serialize, Deserialize, Debug)]
-pub enum Message {
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum ClientToServerMessage {
```

Comment:
TIL: how to implement a simple IPC messaging in Rust using enums.

Reply:
Yeah, I really like this format! 

It's efficient, and if you add a new kind of message the compiler tells you all the places you need to update